examples/lzapp-migration/hardhat.config.ts

```diff
@@ -0,0 +1,93 @@
+// Get the environment configuration from .env file
+//
+// To make use of automatic environment setup:
+// - Duplicate .env.example file and name it .env
+// - Fill in the environment variables
+import 'dotenv/config'
+import 'hardhat-deploy'
+import 'hardhat-contract-sizer'
+import '@nomiclabs/hardhat-ethers'
+import '@layerzerolabs/toolbox-hardhat'
+import { HardhatUserConfig, HttpNetworkAccountsUserConfig } from 'hardhat/types'
+
+// Import LayerZero DevTools functions
+import { EndpointId } from '@layerzerolabs/lz-definitions'
+
+import './tasks/sendV1Message.ts'
+import './tasks/sendV2Message.ts'
```

Comment:
If you import these in `./tasks/index.ts` then you can elide them here, as line 18 will pull them in.